setup.py

```diff
@@ -30,6 +30,13 @@
         'pytest-runner',
     ],
     classifiers=[
-        # TODO(Add items)
+        (
+            'License :: OSI Approved :: ',
+            'GNU General Public License v2 or later (GPLv2+)'
+        ),
```

Comment:
I know we are trying to please flake8, but this is uglier than a long line. The parenthesis are not necessary within a list definition.